src/store/map-settings/index.ts

```diff
@@ -0,0 +1,36 @@
+import { string } from '@recoiljs/refine';
+import { atom } from 'recoil';
+import { urlSyncEffect } from 'recoil-sync';
+
+import { ContextualBasemapsId } from 'types/widget';
+
+export const mapSettingsAtom = atom<boolean>({
```

Comment:
Primitives types (`boolean`, `number`, `string`, ...) are inferred, there's no need to declare them. It's not wrong, just not necessary.